modules/users/src/Volo.Abp.Users.EntityFrameworkCore/Volo/Abp/Users/EntityFrameworkCore/AbpUsersDbContextModelCreatingExtensions.cs

```diff
@@ -16,5 +16,6 @@ public static void ConfigureAbpUser<TUser>(this EntityTypeBuilder<TUser> b)
         b.Property(u => u.EmailConfirmed).HasDefaultValue(false).HasColumnName(nameof(IUser.EmailConfirmed));
         b.Property(u => u.PhoneNumber).HasMaxLength(AbpUserConsts.MaxPhoneNumberLength).HasColumnName(nameof(IUser.PhoneNumber));
         b.Property(u => u.PhoneNumberConfirmed).HasDefaultValue(false).HasColumnName(nameof(IUser.PhoneNumberConfirmed));
+        b.Property(u => u.IsActive).HasDefaultValue(true).HasColumnName(nameof(IUser.IsActive));
```

Comment:
This will cause a problem.
https://github.com/dotnet/efcore/issues/15323#issuecomment-482093707